.travis.yml

```diff
@@ -0,0 +1,81 @@
+language: python
+sudo: required
+
+env:
+  global:
+    - OFFICIAL_REPO="slaclab/timechart"
+    # Doctr deploy key for slaclab/timechart
+    - secure: "EUgO/bmv052rKCsZADTn875axejf7aSgS+IP498YUK/7RqPFSxT+kX2GP9yrcCVMfPokcITLcmXPgrnTjAeYmVwHTdU6V19CeZ0f6uG1CprhVlxKpdgKwI/nuS98XnXfvklxaQu4ANCI5dprSbvxDvVqV0Zkd8IaG8cru5XqkAewYk3+UJot5Q2QnpRmn30juTAD8O0sdLC4Y9EJlwaf9yj1nGvUfryxNJrDES9W/L6w/8Oez1hIQsL/1r9IeU6tksHUkfv+MC/e26yskii5Gbyu2G50w3WY5dzkQI42ggYQeI2kb3a/rka9x4wx1S8eDWpAzZkgKnpnzfJd6bsG5y4yIxfihy98mFbZwWiayFMYNdFf5nud2j8qwrrvXUmTJBQrAlpj8eSDkzoeuutdk4vS3phtUR2OELwTCSf2YEdNxIIBNu/B2l/0aBBW2sEaxIfwQAkMNXPR1KLSw5kqOIVgQ0Hef8+lcdlFE39aiDKJ+iBpgWs33z5IBSk7TC3YMyfbqm3O7YAztca/PIILT0pslsWbSBfJEKIhT/VVWwZ3wrOabuX4LSM17D5ng7dyiF4tJB1L6yJWBrBc6Qa2pOu93zM4j6YDcuenexmKazjaXFUDd9IdUbLIOlv+ZP5Qt9u+OFjpjkL47y8f48VfS+DP2E4hoMJF8AEGMO0eO1s="
+
+matrix:
+  include:
+    - python: 2.7
+      env: PYDM_CHANNEL=pydm-tag
```

Comment:
No need to have the environment variable for `PYDM_CHANNEL` if it is always the same.

Reply:
I'd like to allow flexibility for any future change, so I'll leave the env variable here.